doc/preprocessing.rst

```diff
@@ -69,6 +69,7 @@ Projections:
 
    ICA
    Xdawn
+   annotate_amplitude
    annotate_break
    annotate_flat
```

Comment:
deprecated functions should be removed from the ref. It will also tell us if we need to change any other single-backtick refs to double-backticks/removals